spec/factories/person.rb

```diff
@@ -0,0 +1,14 @@
+FactoryBot.define do
+  factory :person do
+    profile { association(:profile) }
+    buy_intention { association(:buy_intention) }
+    name { Faker::Name.name }
+    social_name { Faker::Name.initials  }
+    email { Faker::Internet.email }
+    cpf { CPF.generate }
+    cnpj { CNPJ.generate }
+    rg { Faker::IDNumber.valid }
+    expediter_rg { Faker::GameOfThrones.house }
+    uf_expediter_rg { Faker::Lorem.characters(2) }
```

Comment:
Take a look at this command "Faker::Address.state_abbr" it generates a state abbreviation